### PR TITLE
[dunfell][gatesgarth][hardknott][master] Regenerate all current releases with updated rosdep and fmi* work around in ros-generate-cache.sh

### PR DESCRIPTION
**Pull request by shr-project**

### CODE REVIEW

File: scripts/ros-generate-cache.sh [hunk 1]
```diff
@@ -115,19 +115,13 @@ cd - > /dev/null
...
 fi
 
-# https://github.com/jediofgever/ROS_CB-release.git doesn't exist anymore and rosdistro_build_cache fails because of that
-# https://github.com/ros/rosdistro/commit/f0193fb9a26b4260fb853e355198f50e3c2d2a4d
-if [ "$ROS_DISTRO" = "foxy" ] ; then
-    sed '/^ *ros_cb:/,/^ *status: developed$/d' -i $ROS_DISTRO/distribution.yaml
-fi
-
-rosdistro_build_cache --preclean --ignore-local $tmpdir/index-v4.yaml $ROS_DISTRO
+rosdistro_build_cache --debug --preclean --ignore-local $tmpdir/index-v4.yaml $ROS_DISTRO
```
Comment (herb-kuta-lge) [inline scripts/ros-generate-cache.sh:124]: Drop `--debug`.

**Review by herb-kuta-lge — CHANGES_REQUESTED** [review]

File: scripts/ros-generate-cache.sh (hunk 1 above)
Comment (shr-project) [reply scripts/ros-generate-cache.sh:124]: Why?

Comment (herb-kuta-lge) [reply scripts/ros-generate-cache.sh:124]: Oh, the commit message explains why it's there. Perhaps add the explanation as a comment here as well?

**Comment by shr-project** [comment]
Merged in other branches as well (except the last commit which was only for dunfell):
   6f4356593..8ebad24b0  dunfell -> dunfell-next
   0539ba2ac..f0414c57b  gatesgarth -> gatesgarth
   0539ba2ac..f0414c57b  gatesgarth -> gatesgarth-next
   4ab7351e3..f295a3a2d  hardknott -> hardknott
   4ab7351e3..f295a3a2d  hardknott -> hardknott-next
   4ab7351e3..f295a3a2d  master -> master
   4ab7351e3..f295a3a2d  master -> master-next

**Pull request merged**